base/dates/periods.jl

```diff
@@ -146,7 +159,6 @@ coarserperiod(::Type{Month}) = (Year,12)
 # and convert more-precise periods to less-precise periods when possible
 """
     CompoundPeriod
-
```

Comment:
same here and below, I think the docstring convention has one blank line between the signature and the body